src/target/mod.rs

```diff
@@ -318,7 +318,7 @@ pub trait Target {
         None
     }
 
-    /// Handle `qOffsets` command
+    /// Handle requests to get the target's current section (or segment) offsets
```

Comment:
Ah... periods -- this is kind of a stylistic thing I do, because I personally think it looks a bit noisy (particularly on struct fields), definitely looks better when rendered with rustdoc though.